package.json

```diff
@@ -16,7 +18,8 @@
     "doc": "doc"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "rollup -c --bundleConfigAsCjs rollup.config.js",
```

Comment:
Changes to `iife` output in a later commit and there's a commit explicitly related to CommonJS, so I guess this might be correct?

Reply:
Yeah, I went back and forth on this for a while, as I didn't quite understand what it was doing. Basically, `--bundleConfigAsCjs` allows us to use CommonJS like `__dirname` or `require.resolve` in the config file itself, even if the file is written in `es` format. I've got rid of it as we don't use any of those so far.

Reply:
Nope, I'm wrong again - we do need it, otherwise we get this when calling `import`:
```
RollupError: Node tried to load your configuration file as CommonJS even though it is likely an ES module. To resolve this, change the extension of your configuration to ".mjs", set "type": "module" in your package.json file or pass the "--bundleConfigAsCjs" flag.
```